src/main/java/Jan.java

```diff
@@ -0,0 +1,96 @@
+import java.util.Scanner;
+
+public class Jan {
+
+    public static final int MAXIMUM_TASK = 100;
```

Comment:
I really like how you defined a constant MAXIMUM_TASK instead of using a magic number! 👍